src/services/storage-options/restApiService.js

```diff
@@ -90,7 +87,10 @@ export default class RestApiService extends StorageInterface {
 
   async getAttachment(attachmentId) {
     const response = await axios.get(
-      `http://localhost:5000/v1/attachments/${attachmentId}/object`
+      `http://localhost:5000/v1/attachments/${attachmentId}/object`,
+      {
+        withCredentials: true,
```

Comment:
You can't hardcode this here because they won't always be using cookie auth. They can still use a bearer token as well. Please use YattIntegrationHelpers.getHeaders.